docs/api.md

```diff
@@ -26,12 +26,18 @@ Forms and their submissions are also accessible through two **open standards spe
 * The [OpenRosa](https://docs.getodk.org/openrosa/) standard allows standard integration with tools like the [ODK Collect](https://docs.getodk.org/collect-intro/) mobile data collection app, or various other compatible tools like [Enketo](https://enketo.org/). It allows them to see the forms available on the server, and to send new submissions to them.
 * The [OData](http://odata.org/) standard allows data to be shared between platforms for analysis and reporting. Tools like [Microsoft Power BI](https://powerbi.microsoft.com/en-us/) and [Tableau](https://public.tableau.com/en-us/s/) are examples of clients that consume the standard OData format and provide advanced features beyond what we offer. If you are looking for a straightforward JSON output of your data, or you are considering building a visualization or reporting tool, this is your best option.
 
-Finally, **system information and configuration** is available via a set of specialized resources. Currently, you may set the Backups configuration, set the Usage Reporting configuration, and retrieve Server Audit Logs.
+Finally, **system information and configuration** is available via a set of specialized resources. Currently, you may set the set the Usage Reporting configuration and retrieve Server Audit Logs.
 
 ## Changelog
 
 Here major and breaking changes to the API are listed by version.
 
+### ODK Central v1.5.3
+
+**Removed**:
+
+* It is no longer possible to initiate a new backups configuration (`POST /v1/config/backups/initiate`) or to verify one (`POST /v1/config/backups/verify`). However, if there is an existing configuration, it is still possible to get it or terminate it. If the existing configuration is terminated, it will not be possible to set up a new configuration. Note that it is still possible to download a [Direct Backup](/reference/system-endpoints/direct-backup). For more information about this change, please see [this topic](https://forum.getodk.org/t/backups-to-google-drive-from-central-will-stop-working-after-oct-3/38895) in the ODK Forum.
```

Comment:
Thanks for doing this @matthew-white. 👍 Any clarity around whether backups will keep happening for existing configurations? That doesn't need to be highlighted here but it would be good to know and to update the forum thread.

Reply:
My best guess is that backups will keep happening for existing configurations, but I'm not sure. Here's what I've learned so far:

- An app may have one or more OAuth clients, which may use one or more redirect URIs. We use a desktop client, which in contrast to a client for web apps, does not seem to require redirect URIs to be preconfigured. [This page](https://support.google.com/cloud/answer/6158849) has more information about different OAuth client types.
- The redirect URI that we use is for the deprecated OOB flow. Again, that's not preconfigured, but it's specified when we set up Google things in Backend ([here](https://github.com/getodk/central-backend/blob/78a75c0dce31b4b897fbcae81d64b6b3ee7628cd/lib/external/google.js#L17)). That said, I believe that the redirect URI is only used to acquire the authorization code that's exchanged for the first set of access and refresh tokens. ([This page](https://developers.google.com/identity/protocols/oauth2/) has more information about the OAuth process as a whole.) In other words, our redirect URI will stop working, which means that it'll stop being possible to acquire a new authorization code, but I haven't seen much to indicate that the parts of the OAuth process that don't use the redirect URI will be affected.
- Just for the record, it doesn't look like there's an easy way for us to specify a different redirect URI. For desktop clients, the OOB migration guide [recommends](https://developers.google.com/identity/protocols/oauth2/resources/oob-migration#desktop-client) a redirect URI for the loopback IP address flow. But @issa-tseng says that that flow wouldn't work us, since Central isn't actually a desktop client.
- At least one Google employee [thinks](https://groups.google.com/g/adwords-api/c/idMbRt1Zg38) that existing tokens will continue to work (but isn't sure). Elsewhere, a different Google employee [responded](https://groups.google.com/g/google-cloud-dev/c/2OvJZehxYYA) to the same question, but they think that existing tokens won't continue to work.
- Another open-source community seems to be dealing with this issue ([here](https://forum.rclone.org/t/google-oauth-migration-for-rclone/30545)). From what I can tell, they believe that existing tokens will continue to work.
- [Some](https://wordpress.org/support/topic/google-oauth-oob-flow-warning/) [folks](https://github.com/r-lib/gargle/issues/202#issuecomment-1147768090) say that they received an email from Google suggesting that an extension of the deadline to January 31, 2023 was possible. That email apparently said, "Keep in mind that all OOB authorization requests will be blocked on February 1, 2023." But to me, that's a pretty different message from something like "Any activity derived from an OOB flow, including access tokens, will be blocked."
- If you try to set up a new backups configuration in Central today, you'll see the warning message [mentioned](https://developers.googleblog.com/2022/02/making-oauth-flows-safer.html#warning-message) in the Google blog post. That message includes the somewhat scary text, "Google will soon block apps that don’t comply with Google’s security policies." Presumably old installations of ODK Central will continue sending OOB authorization requests for a while to come. However, the relevant [help page](https://support.google.com/accounts/answer/12379384) doesn't seem to be saying that the app will be removed from Google Cloud or anything like that. More it just seems like any OOB authorization request will fail.

To me, these sources weigh in the direction of new OOB authorization failing while existing access tokens continue to work. However, I don't feel certain about that.

In terms of next steps:

- If backups will keep happening for existing configurations, this PR should be the only change needed for Backend.
- If backups won't keep happening, we should remove lib/bin/backup.js and probably some other things.
- Either way, we should modify Frontend.
- I also think we should update the forum thread.

I would lean toward the following:

- Tell users that we think backups will keep happening, but aren't sure. It might be reassuring for us to say explicitly that backup restore will continue to work.
- Update Backend/Frontend/`central` along those lines.
- If that turns out to be wrong and everything stops working on October 3, we could probably release another patch that removes more things.

That said, I'd greatly appreciate a second opinion on all this!

Reply:
We ended up receiving a 3-month extension to the deadline: see https://forum.getodk.org/t/backups-to-google-drive-from-central-will-stop-working-after-jan-31st/38895/5.